server/schedulers/balance_leader.go

```diff
@@ -74,6 +75,8 @@ func (l *balanceLeaderScheduler) Schedule(cluster schedule.Cluster, opInfluence
 
 	source := cluster.GetStore(region.Leader.GetStoreId())
 	target := cluster.GetStore(newLeader.GetStoreId())
+	log.Debugf("source store is %v", source)
```

Comment:
please combine two logs into one, add scheduler name .

Reply:
add `[region xxx] xxx`

Reply:
ok, but I think it's not necessary to add scheduler name, cause log will print file name